src/homework6/main_bash.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
I talked to the professor after the lecture the other day, and he suggested us to add sleep 999999 to the bash script, since mariadb might take a while to run.